definition_validator/validator.cc

```diff
@@ -961,6 +961,9 @@ class ValidateWalk {
             variance::validateMethodVariance(ctx, methodDef.symbol);
         }
 
+        // See the comment on `VarianceValidator::validateMethod` for an explanation of why we don't
+        // need to check types on instance variables.
```

Comment:
Uber non-blocking nit: either change this to "See the comment in..." or move the comment inside `validateMethod` outside of the method.

Reply:
I'll queue this up for the followup changes I promised in #6720